syntaxes/test/data/expression.html

```diff
@@ -70,6 +70,7 @@
 {{ condition ? ['123'] : { test: 'a' } }}
 {{ condition ? 'test' : "test" }}
 {{ condition ? [function(), variable] : {} }}
+{{ condition ? condition2 ? condition3 ? 1 : 2 : 3 : 4 }}
```

Comment:
Looks like you need a test for the nullish coalesce case (`??`). Since this test doesn't exist but the case is covered in the regex, I'm curious if you grabbed the regex from another source that we can link to if it's something "official" like the TypeScript repo. You mentioned "TypeScript TM grammar's implementation" in the PR comment - is that something we can link to?

Reply:
Good catch, we should remove that in the first place since the template grammar doesn't have the `??` syntax.